Changelog.md

```diff
@@ -1,9 +1,21 @@
 Changelog
 =========
 
-### 2.1.3 (2018-xx-xx)
+### 2.2.0 (2020-01-08)
 
 * Added Mongolian translation.
+* Added an email prodiver.
+* Added a custom user checker.
+* Added PHP 7.3 support.
+* Removed fieldName attribute in MongoDB mapping.
+* Registration confirmation now redirects to login page if token is invalid.
+* Bumped default minimal password length to 8 characters.
```

Comment:
this is a BC break for existing projects. I think we need to revert that commit and re-apply it only in 3.x

Reply:
Good catch: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2954